db/fixtures/ae_datastore/ManageIQ/Deployment/ContainerProvider/System/StateMachine/Deployment.class/__methods__/analyze_deployment.rb

```diff
@@ -1,37 +1,31 @@
 def analyze_deployment
   $evm.log(:info, "********************** #{$evm.root['ae_state']} ***************************")
-  $evm.root['deployment_method'] = $evm.root['automation_task'].automation_request.options[:attrs][:deployment_method]
-
-  if $evm.root['deployment_method'] == "managed_provision"
+  if $evm.root['deployment_method'] == "provision"
     $evm.root['ae_next_state'] = "provision"
   else
     $evm.root['ae_next_state'] = "check_ssh"
-    $evm.root['deployment_master'] = $evm.root['automation_task'].automation_request.options[:attrs][:deployment_master]
     $evm.root['masters'] = $evm.root['automation_task'].automation_request.options[:attrs][:masters]
     $evm.root['nodes'] = $evm.root['automation_task'].automation_request.options[:attrs][:nodes]
+    $evm.root['deployment_master'] = $evm.root['automation_task'].automation_request.options[:attrs][:deployment_master]
```

Comment:
TLDR :-) Please extract `$evm.root['automation_task'].automation_request.options[:attrs]` (or pat of it) into a variable.